SDRMR/config.yaml

```diff
@@ -22,24 +22,28 @@ arch:
   - i386
 options:
   debug: False
+  rtltcpdebug: False
   msgType: scm
   ids: ""
   duration: 15
   pause_time: 30
   gas_unit_of_measurement: ft³
-  electric_unit_of_measurement: Wh
+  electric_unit_of_measurement: kWh
```

Comment:
By my understanding most meters report in kWh, so I am unsure why we are changing the default?

Reply:
The previous default was Wh not kWh. Like you said, most meters report in kWh